specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -2195,20 +2195,25 @@
           "type": "string",
           "description": "The type of the action.",
           "enum": [
+            "Rotate",
             "rotate",
```

Comment:
k in one case u are adding Rotate while keeping rotate, in the other case you are replacing notify with Notify. why the difference?

Reply:
@rkmanda @adarce was there a reason the old `rotate` value was left in? This is causing us issues consuming as we don't expect valid enum values to vary only by case.

Reply:
@rkmanda Key Vault returned "Notify", not "notify", so that was replaced. Managed HSM doesn't support notify, so there's no camelCase member.

@danielrbradley what are you using to consume? If [autorest](https://github.com/Azure/autorest), it should work. Because Managed HSM and Key Vault share a swagger, adding both Key Vault's "Rotate" and Managed HSM's "rotate" was deemed the least worse option because this behavior has been in the services for around a year. Changing the behavior - which Key Vault did to match the swagger - was a breaking change and was reverted. Its unfortunate, but the service behavior change was much worse. Perhaps you could preprocess the swagger to remove one of the "rotate" values and make sure your enum parsing is case-insensitive for key rotation policy.